cmake/ProjectPaillier.cmake

```
@@ -0,0 +1,31 @@
+include(ExternalProject)
+set(PAILLIER_LIB_SUFFIX a)
+
+ExternalProject_Add(paillier
+    PREFIX ${CMAKE_SOURCE_DIR}/deps
+    DOWNLOAD_NO_PROGRESS 1
+    DOWNLOAD_NAME V1.0.tar.gz
+    URL https://github.com/Shareong/paillier/archive/V1.0.tar.gz
```

Don't recommend use the relased paillier since it hasn't been tested.
```suggestion
    URL https://github.com/Shareong/paillier/archive/V1.0.tar.gz
```
```suggestion
    URL https://github.com/Shareong/paillier/archive/V1.0.tar.gz
```
```suggestion
    URL https://github.com/Shareong/paillier/archive/V1.0.tar.gz
```

please refer to `cmake/ProjectTASSL.cmake` and use `GIT_REPOSITORY ` to depend on the latest paillier : GIT_REPOSITORY https://github.com/jntass/TASSL.git